webapp/config/packages/monolog.yaml

```diff
@@ -0,0 +1,11 @@
+monolog:
+    # Set up logger for the event feed importer.
+    # Note that all other handlers need "!event-feed-importer" in their channels
```

Comment:
It would be nice if we have something which checks if this is indeed the case, but the best I could come up with is doing a grep in CI over all src files which makes sure that the lines with `channels` but without `!event-feed-importer` is only 1.. which sounds ugly.

If we dont have it and we add a new logger we might forget to add this and as shadowing is not used often its easy that it will only show up when you're actually debugging.

Reply:
Which won’t harm anyone, it’s only annoying. But I have an idea, let me try it